services/auth-service/src/main/java/com/sourcefuse/jarc/services/authservice/enums/RoleKey.java

```diff
@@ -0,0 +1,20 @@
+package com.sourcefuse.jarc.services.authservice.enums;
+
+public enum RoleKey {
```

Comment:
no need of putting in numbers here

Reply:
We store numbers in DB hence to map that value here we need numbers